libs/locales.js

```diff
@@ -0,0 +1,40 @@
+const { readdirSync, readFileSync } = require('fs')
+const { join } = require('path')
+const { get } = require('lodash')
+const { io } = require('./panel')
+
+
```

Comment:
Double empty lines? Maybe you want to use ESLint.

Reply:
Yes, i will setup eslint later. Thank you.